### ISSUE TITLE
Line item totals on edit order page should be consistent with email notification

### ISSUE THREAD

**Issue by johndcoy** [woocommerce/woocommerce#23214]
On the edit order page, a product with a price of $10  shows with the discount applied ($8 ) and the difference the coupon made (-$2).

https://cld.wthms.co/aTNBr1

However, the email notification shows a fully priced line item with a separate discount applied. 

https://cld.wthms.co/i4EQCC

The expectation would be for the full price to show per line item in the WP Admin ( as it does in the email notification ) and a possible order subtotal introduced to the WP Admin.

**Comment by mikejolley** [comment]
Admin shows different data to customer facing views.

Notice that in the cart/checkout/view-order discount has it's own totals line - this is the total discount across all items.

In admin, the discount you're referring to is the line discount. It's shown there because it's also editable by admin.

Discounts are applied to line items. Having a separate discount total line in admin would mean you can not associate discounts with line items, which would break tax and other calculations which require the line item costs before and after discounts.

tl;dr The admin needs to be able to edit line item discount values, and in this case that means showing and editing discounts per line, not on the cart as a whole. For the customer, they only need to see a total.

Hope that makes some sense.
